### ISSUE TITLE
[HOLD #10894] [Bug] Attachment - Small image disappears for a brief moment after tap on send

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#10029]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Launch the app and log in NewDot
2. Go to chat with any user
3. On a conversation, click on the + button in the compose box
4. Click on add attachment and upload an image
Small image: 
![161613034-68f73dac-0734-400d-ba3b-c7068995598f (2)](https://user-images.githubusercontent.com/93399543/179972264-9a1bda86-1558-4320-854c-e865b8880d04.jpeg)

## Expected Result:
Small image does not disappear for a brief moment after tap on  send 

## Actual Result:
Small image disappears for a brief moment after tap on send

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.85.5

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):** any

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/179972881-4ee0f4eb-b670-4d91-8906-f8e768f572fe.mp4

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause - Internal Team

**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by techievivek** [comment]
Will look into it today.

**Comment by techievivek** [comment]
Once the `AddAttachment` is refactored, I will re-look into it. From my initial inspection, it looks like for large files, a loading spinner gets rendered, but for very small file sizes it just disappears and reappears.

**Comment by techievivek** [comment]
Not overdue. Its on hold.

**Comment by techievivek** [comment]
On hold, will look into it when things are sorted.

**Comment by techievivek** [comment]
On hold, will be able to look into it only after the API refactor is completed.

**Comment by techievivek** [comment]
On hold, will have a look after the API refactor is done.

**Comment by techievivek** [comment]
Going to pass it to external contributors.

**Comment by mallenexpensify** [comment]
Holding cuz of main tracking issue https://github.com/Expensify/App/issues/10894
Want to wait til Rory's back then ping him to see if we should post the job or now

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by mallenexpensify** [comment]
@mvtglobally can you see if it's reproducible please?

**Comment by mvtglobally** [comment]
@mallenexpensify Issue is no longer repro it seems 

https://user-images.githubusercontent.com/43995119/192202488-c2a9e9f1-7549-4a3f-90ce-00f1899298b5.mp4

**Comment by mallenexpensify** [comment]
Gonna close since it's not reproducible, reopen if needed.  thanks @mvtglobally !
